### PR TITLE
Newsletter-154:Translate into Japanese

### PR DESCRIPTION
**Pull request by azuchi**

### CODE REVIEW

File: _posts/ja/newsletters/2021-06-23-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+  ユーザーが自分のウォレットから使用するトランザクションに署名する方法の選択をユーザーに案内するための別画面が追加されました。
+  ホットウォレットの場合はサーバーが署名するだけですが、鍵が別の場所に保存されているウォレットの場合、
+  魅力的で情報量の多いGUIで、リカバリーニーモニックの入力やハードウェア署名デバイスの使用、
+  署名ウォレットへ転送するPSBTの生成などの署名オプションをユーザーに案内するようになりました。
+
+{% include references.md %}
+{% include linkers/issues.md issues="10823,21365,22156,22144,21261,601,2450,2559" %}
+[LND 0.13.0-beta]: https://github.com/lightningnetwork/lnd/releases/tag/v0.13.0-beta.rc5
+[trw meeting]: https://gist.githubusercontent.com/ariard/5f28dffe82ddad763b346a2344092ba4/raw/2a8e0d4ff431a225a970d0128aa78616df6b6382/meeting-logs
+[additive payment batching]: /en/cardcoins-rbf-batching/
```
Comment (bitschmidty) [inline _posts/ja/newsletters/2021-06-23-newsletter.md:147]: You can link to the /ja/ version now :)

**Review by bitschmidty — CHANGES_REQUESTED** [review]
@azuchi looks good with one suggestion

File: _posts/ja/newsletters/2021-06-23-newsletter.md (hunk 1 above)
Comment (azuchi) [reply _posts/ja/newsletters/2021-06-23-newsletter.md]: @bitschmidty thanks! I fixed it.

**Review by bitschmidty — APPROVED** [review]
@azuchi thanks for the translation again!

**Pull request merged**